src/vue3-app/src/services/ResourceService.ts

```
@@ -16,6 +15,25 @@ class ResourceService<T extends Resource, CreateDTO = Omit<T, 'id'>> {
     this.resource = resource;
   }
 
+  private static instanceMap: { [key: string]: ResourceService<any, any> } = {};
+
+  static getInstance<T extends Resource, CreateDTO = Omit<T, 'id'>>(
+    resource: string
+  ): ResourceService<T, CreateDTO> {
+    if (!this.instanceMap[resource]) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)